dist/pom.xml

```diff
@@ -51,25 +51,148 @@
     <profiles>
         <profile>
             <id>noSnapshotsWithDatabricks</id>
-            <properties>
-                <included_buildvers>
-                    ${noSnapshot.buildvers},
-                    ${databricks.buildvers}
-                </included_buildvers>
-            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <executions>
+                            <execution>
+                                <id>set-included-buidlvers</id>
+                                <goals><goal>run</goal></goals>
+                                <phase>generate-sources</phase>
+                                <configuration>
+                                    <target xmlns:ac="antlib:net.sf.antcontrib">
+                                        <exec executable="python" outputproperty="included_buildvers">
```

Comment:
is it necessary to rely on external python process? We have previously used jython for portability https://github.com/NVIDIA/spark-rapids/blob/d2abcd9d8e8cd103273e8e8db4fa24d2ef949046/dist/pom.xml#L368